src/libraries/System.Formats.Asn1/tests/System.Formats.Asn1.Tests.csproj

```diff
@@ -0,0 +1,51 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TargetFrameworks>$(NetCoreAppCurrent);$(NetFrameworkCurrent)</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup><!--
```

Comment:
I assume this commenting out is so that tests would pass on this interim commit?  I guess I'll find that out as I review subsequent commits.  Ah the suspense :)